
[Pytest Session Setup] Попытка сброса последовательностей ID...
Не удалось подключиться к БД BRT для сброса последовательностей.
